Content.Server/Chemistry/ReagentEffects/PlantMetabolism/PlantSedin.cs

```diff
@@ -0,0 +1,49 @@
+using Content.Server.Botany.Components;
+using Content.Server.Botany.Systems;
+using Content.Shared.Chemistry.Reagent;
+using JetBrains.Annotations;
+using Content.Server.Popups;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Random;
+
+namespace Content.Server.Chemistry.ReagentEffects.PlantMetabolism
+{
```

Comment:
I think we should be using file-scoped namespaces for new files, as mentioned in the contribution guidelines, on bullet point 2 under file-layout
https://docs.spacestation14.com/en/general-development/codebase-info/conventions.html#file-layout

Reply:
It seems that somehow I've reviewed an older commit, the latest commit seems to have fixed this already, so this one can be resolved.